sdk/examples/client/ledger_nano_transaction.rs

```diff
@@ -23,7 +23,7 @@ use iota_sdk::client::{
     Client, Result,
 };
 
-const AMOUNT: u64 = 1_000_000;
+// const AMOUNT: u64 = 1_000_000;
```

Comment:
?

Reply:
It's because I rebased from develop, some examples weren't even compiled, another part of the code was commented but not this so it wasn't compiling